src/CoreText/CTStringAttributes.cs

```diff
@@ -120,6 +121,7 @@ static CTStringAttributeKey ()
 				GlyphInfo                   = Dlfcn.GetStringConstant (handle, "kCTGlyphInfoAttributeName");
 				CharacterShape              = Dlfcn.GetStringConstant (handle, "kCTCharacterShapeAttributeName");
 				RunDelegate                 = Dlfcn.GetStringConstant (handle, "kCTRunDelegateAttributeName");
+				Offset                      = Dlfcn.GetStringConstant (handle, "kCTBaselineOffsetAttributeName");
```

Comment:
I think this should be named `BaselineOffset` (and the same with the public property below).